tests/v2/validation/provisioning/ssh.go

```diff
@@ -0,0 +1,33 @@
+package provisioning
+
+// This file contains all tests that require to ssh into a node to run commands to check things
+// such as any stats, benchmarks, etc. For example, ssh is required to check the cpu usage of a process running on an individual node.
```

Comment:
nit - can you put `process running on an individual node` on its own line? Just so that we don't overcrowd comments on one particular line.